src/com/lftechnology/java/training/alina/basic/calculategpa/Student.java

```diff
@@ -14,98 +14,89 @@
  */
 public class Student extends Course {
 
-	static ArrayList<Student> students = new ArrayList<Student>();
-	static ArrayList<Course> courses = new ArrayList<Course>();
+    static ArrayList<Student> students = new ArrayList<Student>();
+    static ArrayList<Course> courses = new ArrayList<Course>();
 
-	private static final Logger LOGGER = Logger.getLogger(Student.class
-			.getName());
-	private String fullName;
-	private int standard;
-	private int rollNo;
+    private static final Logger LOGGER = Logger.getLogger(Student.class.getName());
+    private String fullName;
+    private int standard;
+    private int rollNo;
 
-	public Student(float subject1, float subject2, float subject3,
-			float subject4, float subject5, String fullName, int standard,
-			int rollNo) {
-		super(subject1, subject2, subject3, subject4, subject5);
-		this.fullName = fullName;
-		this.standard = standard;
-		this.rollNo = rollNo;
-	}
+    public Student(float subject1, float subject2, float subject3, float subject4, float subject5, String fullName, int standard, int rollNo) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Constructor has 8 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00107)